json-rpc/src/runtime.rs

```diff
@@ -484,7 +512,7 @@ fn set_response_error(
             _ => "unexpected_code",
         };
         counters::INVALID_REQUESTS
-            .with_label_values(&[request_type, method, label])
+            .with_label_values(&[request_type, method, label, &sdk_lang.as_str()])
```

Comment:
On all of these calls you don't need the `&` just do: 
```suggestion
            .with_label_values(&[request_type, method, label, sdk_lang.as_str()])
```